docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -277,6 +281,54 @@ fixture `MyFixture`
     .page `../my-project/index.html`;
 ```
 
+## Specifying Testing Metadata
+
+TestCafe allows you to specify additional information for fixtures and tests in the form of *field-value metadata*.
+
+### Fixture Metadata
+
+You can specify metadata for a fixture by using the `meta` method in the [fixture declaration](#fixtures).
+
+```js
+fixture.meta({ field1: 'value1', field2: 'value2', field3: 'value3' });
```

Comment:
Method has an additional overload:
`.meta(name, value)`

Parameter  | Type   | Description
---------- | ------ | -----------------
`name` | string | Name of the metadata entry
`value` | string | Value of the metadata entry